README.rst

```diff
@@ -1,4 +1,4 @@
-Girder Worker |build-status| |docs-status| |license-badge| |gitter-badge|
+|logo| Girder Worker |build-status| |docs-status| |license-badge| |gitter-badge|
```

Comment:
Great! Note that this caused formatting problems before on PyPi: https://github.com/girder/girder_worker/commit/e4543802ea64ea0fa34fd1864d6f4dd746542f18